<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DestroyDialog</class>
 <widget class="QDialog" name="DestroyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>556</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>118</y>
      <width>540</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <widget class="QLabel" name="contactLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>销毁资产</string>
     </property>
    </widget>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>销毁金额:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>合约地址:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="mortgageLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="contractAddressLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="amountLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="assetNameLabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>合约余额:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>当前准备金率:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="contractBalanceLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>账户余额:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="balanceLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="assetNameLabel2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>140</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>执行账户:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="accountLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>12</y>
       <width>13</width>
       <height>13</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
